src/hint_cli.py

```diff
@@ -56,9 +60,51 @@ def parse(argv=None):
     return path, config, action, args
 
 
+def path_last_deploy(path):
+    return path + "/.last_deploy"
+
+
+def save_config(path, config, cfg):
+    dat = {"config": config,
+           "time": time.time(),
+           "data": cfg}
+    with open(path_last_deploy(path), "wb") as f:
+        pickle.dump(dat, f)
+
+
+def read_config(path):
+    with open(path_last_deploy(path), "rb") as f:
+        dat = pickle.load(f)
+    return dat
+
+
+def load_config(path, config=None, refresh=True):
+    if os.path.exists(path_last_deploy(path)):
+        dat = read_config(path)
+        if refresh:
+            print("[Reloaded configuration '{}' ({} s old)]".format(
+                dat["config"] or "<base>", round(time.time() - dat["time"])))
+            cfg = HintConfig(path, dat["config"])
+        else:
+            print("[Loaded configuration '{}' ({} s old)]".format(
+                dat["config"] or "<base>", round(time.time() - dat["time"])))
+            cfg = dat["data"]
+    else:
+        cfg = HintConfig(path, config)
+    return cfg
+
+
+def remove_config(path):
+    p = path_last_deploy(path)
+    if os.path.exists(p):
+        print("Removing configuration")
+        os.unlink(p)
+
+
 def main(argv=None):
     path, config, action, args = parse(argv)
-    cfg = HintConfig(path, config)
+    refresh = action in ["start", "user"]
```

Comment:
So you never want to restart with an existing config, but always refresh? I thought that was the idea?
Why does 'user' need a refreshed config?

Reply:
The update was going to be in the next PR but as noted above, this is a complication we don't need and I've removed it